pkg/plugins/external/webhook.go

```diff
@@ -35,6 +36,24 @@ func (p *createWebhookSubcommand) InjectResource(*resource.Resource) error {
 	return nil
 }
 
+func (p *createWebhookSubcommand) BindFlags(fs *pflag.FlagSet) {
+	req := external.PluginRequest{
+		APIVersion: defaultAPIVersion,
+		Command:    "flags",
+		Args:       []string{"--webhook"},
+	}
+
+	// Get a list of flags for the init subcommand of the external plugin
+	// If it returns an error, parse all flags passed by the user and let
+	// the external plugin return an unknown flag error.
+	flags, err := getExternalPluginFlags(req, p.Path)
+	if err != nil {
+		bindAllFlags(fs, p.Args)
+	} else {
+		bindSpecificFlags(fs, flags)
+	}
+}
```

Comment:
The code changes here are equal for all places
Could we find a way of not duplicate/

Reply:
Should be resolved in https://github.com/kubernetes-sigs/kubebuilder/pull/2693/commits/88a68a587f85126c4b5b68bec28a7bfa881456ad